ci/travis/install-dependencies.sh

```diff
@@ -121,6 +123,35 @@ run_npm() {
   npm ci
 }
 
+install_npm_project() {
+  if [ "${OSTYPE}" = msys ]; then
+    # Not Windows-compatible: https://github.com/npm/cli/issues/558#issuecomment-584673763
+    { echo "WARNING: Skipping running NPM due to module incompatibilities with Windows"; } 2> /dev/null
+  else
+    run_npm
+  fi
+}
+
+# Takes a list of pip requirements and prints the ones that don't have known incompatibilities with the current system.
+filter_compatible_pip_requirements() {
```

Comment:
Let's remove this. We can for example address it later by trying to remove the dependencies that are incompatible.

Reply:
Wouldn't it just break something if I remove them from the requirements file? I think one of them was [`blist`](https://github.com/DanielStutzbach/blist/issues/81#issue-391460716), I forget if there was another one. Should I just delete those lines from the requirements? I would've guessed it would break something for users.